### DISCUSSION TITLE
AttributeError: module 'pymonctl' has no attribute 'getSize'

### DISCUSSION THREAD

**Discussion by JamesOwenB** [tryptech/obs-zoom-and-follow#130] [Q&A]
[zoom_and_follow_mouse.py] Traceback (most recent call last):
[zoom_and_follow_mouse.py]   File "C:\Program Files/obs-studio/data/obs-plugins/frontend-tools/scripts\zoom_and_follow_mouse.py", line 1097, in script_properties
[zoom_and_follow_mouse.py]     populate_list_property_with_monitors(m)
[zoom_and_follow_mouse.py]   File "C:\Program Files/obs-studio/data/obs-plugins/frontend-tools/scripts\zoom_and_follow_mouse.py", line 915, in populate_list_property_with_monitors
[zoom_and_follow_mouse.py]     screen_size = zoom.monitors_dict.get(monitor, None)['size'] if darwin else pmc.getSize(monitor)
[zoom_and_follow_mouse.py] AttributeError: module 'pymonctl' has no attribute 'getSize'
[zoom_and_follow_mouse.py] Traceback (most recent call last):
[zoom_and_follow_mouse.py]   File "C:\Program Files/obs-studio/data/obs-plugins/frontend-tools/scripts\zoom_and_follow_mouse.py", line 1097, in script_properties
[zoom_and_follow_mouse.py]     populate_list_property_with_monitors(m)
[zoom_and_follow_mouse.py]   File "C:\Program Files/obs-studio/data/obs-plugins/frontend-tools/scripts\zoom_and_follow_mouse.py", line 915, in populate_list_property_with_monitors
[zoom_and_follow_mouse.py]     screen_size = zoom.monitors_dict.get(monitor, None)['size'] if darwin else pmc.getSize(monitor)
[zoom_and_follow_mouse.py] AttributeError: module 'pymonctl' has no attribute 'getSize'
[zoom_and_follow_mouse.py] Traceback (most recent call last):
[zoom_and_follow_mouse.py]   File "C:\Program Files/obs-studio/data/obs-plugins/frontend-tools/scripts\zoom_and_follow_mouse.py", line 1097, in script_properties
[zoom_and_follow_mouse.py]     populate_list_property_with_monitors(m)
[zoom_and_follow_mouse.py]   File "C:\Program Files/obs-studio/data/obs-plugins/frontend-tools/scripts\zoom_and_follow_mouse.py", line 915, in populate_list_property_with_monitors
[zoom_and_follow_mouse.py]     screen_size = zoom.monitors_dict.get(monitor, None)['size'] if darwin else pmc.getSize(monitor)
[zoom_and_follow_mouse.py] AttributeError: module 'pymonctl' has no attribute 'getSize'

**Comment by oldirtyewok** [comment]
I have the same issue.
OBS 29.1.3 (64bit)
Win 11
2 Monitors (4k + 1080p)
Python 3.10.9 (also tried with 3.10.11)

**Reply by asprem** [reply]
I am getting this error.
Installed Pymongo as well but still have ing an issue

[zoom_and_follow_mouse.py] Traceback (most recent call last):
[zoom_and_follow_mouse.py]   File "C:\My Data/My Edits/OBS/obs-zoom-and-follow-master/obs-zoom-and-follow-master\zoom_and_follow_mouse.py", line 7, in <module>
[zoom_and_follow_mouse.py]     import pymonctl as pmc
[zoom_and_follow_mouse.py] ModuleNotFoundError: No module named 'pymonctl'
[zoom_and_follow_mouse.py] Traceback (most recent call last):
[zoom_and_follow_mouse.py]   File "C:\My Data/My Edits/OBS/obs-zoom-and-follow-master/obs-zoom-and-follow-master\zoom_and_follow_mouse.py", line 7, in <module>
[zoom_and_follow_mouse.py]     import pymonctl as pmc
[zoom_and_follow_mouse.py] ModuleNotFoundError: No module named 'pymonctl'
